lib/SILOptimizer/Mandatory/TFDifferentiation.cpp

```diff
@@ -3439,7 +3464,7 @@ class AdjointEmitter final : public SILInstructionVisitor<AdjointEmitter> {
     // TODO: Change this to a note when we emit an error at the @autodiff
```

Comment:
Can this TODO note be removed?

Reply:
Good catch! I think I’ll get rid of em in my next PR since CI has passed.